docs/nextflow_schema/create_schema.md

```diff
@@ -48,7 +48,7 @@ nf-core schema build
 
 !!! warning
 
-    The current version of `nf-core` tools (v2.12.1) does not support the new schema draft used in `nf-validation`. Running this command after building the schema will convert the schema to the right draft:
+    The current version of `nf-core` tools (v2.13.1) does not support the new schema draft used in `nf-schema`. Running this command after building the schema will convert the schema to the right draft:
```

Comment:
That uses the correct draft, are we sure there are no other quirks unqiue to older drafts that the builder relies upon? (maybe not a huge deal really as hopefully `tools` will be updated ASAP)

Reply:
If the schema contains a lot of special quirks, it could be that some other stuff breaks but this should be fine for 99% of the pipelines.